octoprint_display_panel/__init__.py

```diff
@@ -932,9 +691,36 @@ def log_error(self, ex):
 		Helper function for more complete logging on exceptions
 		"""
 
-		template = "An exception of type {0} occurred on {1}. Arguments: {2!r}"
-		message = template.format(type(ex).__name__, inspect.currentframe().f_code.co_name, ex.args)
-		self._logger.warn(message)
+		#template = "An exception of type {0} occurred on {1}. Arguments: {2!r}"
```

Comment:
Reverted to the original version. Eventually it may make sense to replace all the calls to `log_error()` with `self._logger.exception(message)` which is a nice way to log the full traceback.